client/jetpack-cloud/sections/sidebar-navigation/jetpack-manage.tsx

```diff
@@ -56,6 +57,15 @@ const JetpackManageSidebar = ( { path }: { path: string } ) => {
 				menu_item: 'Jetpack Cloud / Plugins',
 			},
 		} ),
+		createItem( {
+			icon: tag,
+			path: '/',
+			link: JETPACK_MANAGE_PRICING_LINK,
+			title: translate( 'Pricing' ),
+			trackEventProps: {
+				menu_item: 'Jetpack Cloud / Partner Portal / Issue License',
```

Comment:
Should we name this `Jetpack Cloud / Partner Portal / Pricing`? We could use the same one when we add a new pricing page.

Reply:
Honestly, I am not really sure which one is correct, but I think you are right, and it makes more sense to use "Pricing" here instead of "Issue License."